File: 110algoritmos/exercicios018/ex018-Csharp/ex018.cs
using System;

class Program {
    static void Main() {
        double angulo, coseno, seno, tangente;

        Console.WriteLine("Digite o angulo:");
        angulo = double.Parse(Console.ReadLine());

        seno = Math.Sin((angulo * Math.PI) / 180.0);
        coseno = Math.Cos((angulo * Math.PI)/180.0);
        tangente = Math.Tan((angulo * Math.PI)/180.0);

        Console.WriteLine("O angulo " + angulo + " tem SENO: " + seno.ToString("F"));
        Console.WriteLine("O angulo " + angulo + " tem COSENO: " + coseno.ToString("F"));
        Console.WriteLine("O angulo " + angulo + " tem TANGENTE: " + tangente.ToString("F"));

    }
}

File: 110algoritmos/exercicios018/ex018-Python/ex018.py
import math

angulo = (float(input("Digite o ângulo que você deseja: ")))
seno = math.sin(math.radians(angulo))
cos = math.cos(math.radians(angulo))
tangente = math.tan(math.radians(angulo))
print(f"\nO ângulo de {angulo} tem o SENO de {seno:.2}")
print(f"\nO ângulo de {angulo} tem o COSENO de {cos:.2}")
print(f"\nO ângulo de {angulo} tem o TANGENTE de {tangente:.2}")